README.md

```diff
@@ -344,3 +346,41 @@ search.addWidget(
   })
 );
 ```
+
+### rangeSlider
+
+![Example of the rangeSlider widget][rangeSlider]
+
+#### API
+
+```js
+/**
+ * Instantiate a slider based on a numeric attribute
+ * @param  {String|DOMElement} options.container Valid CSS Selector as a string or DOMElement
+ * @param  {String} options.facetName Name of the attribute for faceting
+ * @param  {Boolean|Object} [options.tooltips=true] Should we show tooltips or not.
```

Comment:
> What is the default tooltip message if I use only true?

We just show the formatted tooltip value. So in the current case it would be "5.99" instead of "$5.99"